CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ permalink: /docs/en-US/changelog/
 * Always set the database root user password to avoid having the default invalid password on fresh installs
 * Don't spider and recurse domains when checking for network connections
 * Swap the MariaDB apt mirror used for a more reliable source
+* Fix issues with the bear reporting failed provisioners when everything seem to have gone ok ( #2161 )
```

Comment:
```suggestion
* Fix issues with the sad bear showing at the end of provisioning despite provisioners being succesful( #2161 )
```

Reply:
This PR also lists relevant log files which is an enhancement that needs adding further up the file